cpp/src/arrow/compute/kernels/vector_sort_benchmark.cc

```diff
@@ -130,6 +164,18 @@ static void ArraySortIndicesBool(benchmark::State& state) {
   ArraySortFuncBoolBenchmark(state, SortRunner(state));
 }
 
+static void ArraySortIndicesString(benchmark::State& state) {
+  const auto min_length = 0;
+  const auto max_length = 32;
```

Comment:
Define as global const and remove duplicates?

Reply:
No strong preference, though I think this way if we wanted to try more combinations of benchmarks in the future it would be easier to extend